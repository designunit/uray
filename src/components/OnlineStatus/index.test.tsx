import { shallow } from 'enzyme'
import { OnlineStatus } from '.'
import { Icon } from 'antd'

describe('<OnlineStatus/>', () => {
    it('properly render component', () => {
        const component = shallow((
            <OnlineStatus
                status={'online'}
            />
        ))

        expect(component).toMatchSnapshot()
    })
    
    it('should contain <Icon/> component', () => {
        const component = shallow((
            <OnlineStatus
                status={'online'}
            />
        ))

        expect(component.find(Icon).length).toBe(1)
    })
})
