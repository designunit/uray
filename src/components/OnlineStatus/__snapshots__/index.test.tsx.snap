// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<OnlineStatus/> properly render component 1`] = `
<div>
  <Icon
    theme="twoTone"
    twoToneColor="#52c41a"
    type="bulb"
  />
</div>
`;
